content/SCALETutorials/Network/ManagingGlobalConfig.md

```diff
@@ -44,16 +44,23 @@ Go to **Network**, and click **Settings** on the **Global Configuration** widget
 4. Enter the IP address for your default gateway into the **IPv4 Default Gateway** if you are using IPv4 IP addresses.
    Enter the IPv6 address in the **IPv6 Default Gateway** if you are using IPv6 addresses.
 
-5. Select **Allow All** for **Outbound Network**  to permit external communication for all TrueNAS services or select **Deny All** to prevent external communication.
-   Select **Allow Specific** and then use the dropdown list to pick a limited set of services to allow external communication.
+5. Select the desired **Outbound Network** option.
+
+   * Selecting **Allow All** permits external communication for all TrueNAS services.
+
+   * Selecting **Deny All** prevents external communication.
+   
+   * Selecting **Allow Specific** shows the **Services** dropdown where you can choose a limited set of services to allow external communication. Unchecked services cannot communicate externally.
+   
+   * Selecting **Allow All Except** and choosing options on the **Services** dropdown list denies those services external communication while allowing it for all other services. Selected services cannot communicate externally.
```

Comment:
While not incorrect this deviates from the parallel construction in the procedure. Suggest these changes:
```suggestion
   * Select **Allow All** to permit external communication for all TrueNAS services.

   * Select **Deny All** to prevent external communication.
   
   * Select **Allow Specific** to show the **Services** dropdown, and then choose a limited set of services to allow external communication. Unchecked services cannot communicate externally.
   
   * Select **Allow All Except** and choose options on the **Services** dropdown list to deny those services external communication. Selected services cannot communicate externally.
```